UI/css/style.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: inherit;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    box-sizing: border-box;
+    font-family: "Montserrat", sans-serif;
+    font-size: 1.6rem;
+    overflow: hidden;
+}
+
+.navigation {
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    background-color: rgba(255, 255, 255, 0.8);
+    box-shadow: 0 .2rem 2rem rgba(0, 0, 0, 0.25);
+    z-index: 500;
```

Comment:
why is z-index having such big value?

Reply:
I just putted it like to verflow those three bar image.

Reply:
okay let start the mobile responsive part